examples/Cargo.toml

```diff
@@ -30,13 +30,18 @@ path = "compute/main.rs"
 name = "mesh-shading"
 path = "mesh-shading/main.rs"
 
+[[bin]]
+name = "ray-tracing"
+path = "ray-tracing/main.rs"
+
 [dependencies]
 image = "0.23.12"
 log = "0.4"
 hal = { path = "../src/hal", version = "0.6", package = "gfx-hal" }
 auxil = { path = "../src/auxil/auxil", version = "0.5", package = "gfx-auxil" }
 gfx-backend-empty = { path = "../src/backend/empty", version = "0.6" }
 winit = { version = "0.23", features = ["web-sys"] }
+wavefront_obj = "9.0"
```

Comment:
would you mind using https://crates.io/crates/obj instead? it's something we have more control of

Reply:
I don't technically need to load an `obj` file at all for this change, this is just an easy way to get a bunch of data into the API to (hopefully) smoke test it a bit better than just a cube.

Ideally I'll do like a cornell box or something, which seems reasonable to do programmatically and should show off ray tracing pretty well (or maybe just a triangle?).

Reply:
That may end up being a whole lot of code. I.e. you'd need something like `genmesh`, and debugging would be even more difficult. I think loading an `obj` is perfectly reasonable here.

Reply:
Sounds good. Above all else, I want it to be understandable 😁